terraform/locals.tf

```diff
@@ -51,7 +51,7 @@ locals {
     },
     {
       name  = "PLEK_SERVICE_SIGNON_URI"
-      value = "http://signon.tariff.internal"
+      value = "http://signon.tariff.internal:8080"
```

Comment:
```suggestion
      value = "http://signon.tariff.internal:8080/"
```

Reply:
Do we need to append this or does the routing deal with this?

Reply:
I'll test it in a minute but I think it should be fine as long as it routes to the app